src/main/java/org/stellar/sdk/StrKey.java

```diff
@@ -177,20 +229,20 @@ public static String encodeSha256Hash(byte[] data) {
             ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
             outputStream.write(versionByte.getValue());
             outputStream.write(data);
-            byte payload[] = outputStream.toByteArray();
-            byte checksum[] = StrKey.calculateChecksum(payload);
+            byte[] payload = outputStream.toByteArray();
+            byte[] checksum = StrKey.calculateChecksum(payload);
             outputStream.write(checksum);
-            byte unencoded[] = outputStream.toByteArray();
+            byte[] unencoded = outputStream.toByteArray();
 
             if (VersionByte.SEED != versionByte) {
-                return StrKey.base32Encoding.encode(unencoded).toCharArray();
+                return base32Encoding.encode(unencoded).toCharArray();
             }
 
             // Why not use base32Encoding.encode here?
             // We don't want secret seed to be stored as String in memory because of security reasons. It's impossible
             // to erase it from memory when we want it to be erased (ASAP).
             CharArrayWriter charArrayWriter = new CharArrayWriter(unencoded.length);
-            OutputStream charOutputStream = StrKey.base32Encoding.encodingStream(charArrayWriter);
+            OutputStream charOutputStream = base32Encoding.encodingStream(charArrayWriter);
```

Comment:
Errm, note the comment above this line? I think there's a reason we use `StrKey`? Maybe @tamirms knows more on this (or `git blame` it)

Reply:
this comment was referring to the usage of `.encode()` vs `.encodingStream()`, the change here was just source tidy-up, `StrKey.base32Encoding` was redundant, as within `StrKey` class scope, it references the same static member declared here at [base32Encoding](https://github.com/stellar/java-stellar-sdk/pull/413/files/49b477d5c24682dbe3dfa52fed0f9882a3c38d36#diff-d3b42d063d6edd65d7df86ba56cad3a838bffa1662d0de914d563c6eec6f9da0R56)